src/WebJobs.Script.Abstractions/Description/FunctionMetadata.cs

```diff
@@ -42,6 +43,11 @@ public FunctionMetadata()
         /// </summary>
         public IDictionary<string, object> Properties { get; }
 
+        /// <summary>
+        /// Gets or sets the optional function execution retry strategy on invocation failures.<see cref="Abstractions.Description.Retry"/>.
```

Comment:
nit: "Gets or sets the optional function execution retry strategy to use on invocation failures.<see cref="Abstractions.Description.Retry"/>"